data/resources/Urban_Jungle/terrain-graphics.cfg

```diff
@@ -1,6 +1,4 @@
-#textdomain wesnoth-ei
-
-# NOTE - make sure this matches up with terrain.cfg
+# if I change this, make sure it still matches up with terrain.cfg
```

Comment:
Just wondering why this remark was changed to first-person. Who's the 'I' here? The original seems to make more sense to me.

Reply:
Probably something to do with slightly different versions of this file from my various campaigns. I've changed it back to the old "NOTE" remark.

Reply:
I didn't realise this was your file - makes sense!